contracts/cw-ics721-bridge/src/contract.rs

```diff
@@ -199,17 +169,17 @@ fn do_receive_nft(
     let UniversalNftInfoResponse { token_uri, .. } = deps.querier.query_wasm_smart(
         info.sender,
         &cw721::Cw721QueryMsg::NftInfo {
-            token_id: token_id.clone(),
+            token_id: token_id.clone().into(),
         },
     )?;
 
     let ibc_message = NonFungibleTokenPacketData {
         class_id: class_id.clone(),
         class_uri,
+        class_data: None, // TODO: check for values we need to forward from a source chain.
```

Comment:
How come its always none?

Reply:
that's just a temporary state pending the TODO. @Art3mix is going to add the data forwarding so that if we receive data for a token it'll get passed along here.